test-e2e-slow/build-top-100-opam.test.js

```diff
@@ -14,6 +14,7 @@ const rmSync = require('rimraf').sync;
 const isCi = require('is-ci');
 
 let cases = [
+  {name: 'libtorch', toolchains: [ocamlVersion]},
```

Comment:
:+1:

Reply:
How to run slow tests on CI?

Reply:
By adding a commit containing `@slowtest` in it